Analysis/HiggsTauTau/src/HTTWeights.cc

```diff
@@ -2259,6 +2229,7 @@ namespace ic {
       }
     }
 
+
     return 0;
   }
 
```

Comment:
I don't think you meant to change this file, right? Can you merge the master branch in to update to the correct version?